odoo/addons/base/models/ir_qweb.py

```diff
@@ -646,12 +646,12 @@ def _get_template(self, template, options):
             element = template
             document = etree.tostring(template, encoding='unicode')
             ref = None
-
+        # template is falsy
+        elif not template:
+            raise ValueError('Template identifier is falsy')
         # template is xml as string
-        elif isinstance(template, str) and template.startswith('<'):
-            element = etree.fromstring(template)
-            document = template
-            ref = None
+        elif isinstance(template, str) and '<' in template:
+            raise ValueError(f"Using a template as a string instead of etree object:\n{template!r}")
```

Comment:
the error message isn't really clear to me. The point is to ask them to switch to `etree` objects, right?

```suggestion
            raise ValueError("Inline templates must be passed as `etree` documents")
```